common/content/tabs.js

```diff
@@ -49,10 +49,10 @@ const Tabs = Module("tabs", {
     },
 
     /**
-     * @property {Iterator(Object)} A genenerator that returns all browsers
+     * @return {Generator} A genenerator that returns all browsers
      *     in the current window.
      */
-    get browsers() {
+    browsers: function* () {
```

Comment:
Yeah. `browsers` can create generator within property to keep getter, I'm going to try.